Source/SwiftLintBuiltInRules/Rules/Style/OpeningBraceRule.swift

```diff
@@ -169,73 +225,1044 @@ struct OpeningBraceRule: CorrectableRule, ConfigurationProviderRule {
+            class Rule<T>  ↓{
+
+            }
+            """):
+                Example("""
+            class Rule<T> {
+
+            }
+            """),
+            Example("""
+            class Rule<T>: NSObject  ↓{
+
+            }
+            """):
+                Example("""
+            class Rule<T>: NSObject {
+
+            }
+            """),
+            Example("""
+            class Rule<T>: NSObject where T: U  ↓{
+
+            }
+            """):
+                Example("""
+            class Rule<T>: NSObject where T: U {
+
+            }
+            """),
+            Example("""
+            enum Rule
+            ↓{
+
+            }
+            """):
+                Example("""
+            enum Rule {
+
+            }
+            """),
+            Example("""
+            enum Rule: E  ↓{
+
+            }
+            """):
+                Example("""
+            enum Rule: E {
+
+            }
+            """),
+            Example("""
+            extension Rule
+            ↓{
+
+            }
+            """):
+                Example("""
+            extension Rule {
+
+            }
+            """),
+            Example("""
+            protocol Rule  ↓{
+
+            }
+            """):
+                Example("""
+            protocol Rule {
+
+            }
+            """),
+            Example("""
+            struct Rule
+            ↓{
+
+            }
+            """):
+                Example("""
+            struct Rule {
+
+            }
+            """),
+            Example("""
+            struct Rule  : A
+            ↓{
+
+            }
+            """):
+                Example("""
+            struct Rule  : A {
+
+            }
+            """),
+            Example("""
+            do {
+
+            } catch
+            ↓{
+
+            }
+            """):
+                Example("""
+            do {
+
+            } catch {
+
+            }
+            """),
+            Example("""
+            do {
+
+            } catch MyError.unknown  ↓{
+
+            }
+            """):
+                Example("""
+            do {
+
+            } catch MyError.unknown {
+
+            }
+            """),
+            Example("""
+            do {
+
+            } catch let error  ↓{
+
+            }
+            """):
+                Example("""
+            do {
+
+            } catch let error {
+
+            }
+            """),
+            Example("""
+            defer  ↓{
+
+            }
+            """):
+                Example("""
+            defer {
+
+            }
+            """),
+            Example("""
+            do  ↓{
+
+            }
+            """):
+                Example("""
+            do {
+
+            }
+            """),
+            Example("""
+            for a in b
+            ↓{
+
+            }
+            """):
+                Example("""
+            for a in b {
+
+            }
+            """),
```

Comment:
The common indentation style for corrected examples in SwiftLint would be like:
```suggestion
                Example("""
                for a in b {

                }
                """),
```